pyproject.toml

```diff
@@ -44,6 +45,7 @@ classifiers = [
 [project.optional-dependencies]
 tests = [
     "pytest >=6.0",
+    "pytest-split",
```

Comment:
How does `pytest-split` relate to `pytest-xdist` (https://github.com/pytest-dev/pytest-xdist)?